examples/all-clusters-app/ameba/main/chipinterface.cpp

```diff
@@ -142,6 +142,23 @@ static void InitServer(intptr_t context)
 #endif
 }
 
+extern "C" bool matter_server_is_commissioned()
+{
+    return (chip::Server::GetInstance().GetFabricTable().FabricCount() != 0);
+}
+
+extern "C" void matter_get_fabric_indexes(uint16_t *pFabricIndexes)
+{
+    size_t i = 0;
+    for (auto it = chip::Server::GetInstance().GetFabricTable().begin();
+        it != chip::Server::GetInstance().GetFabricTable().end(); ++it)
```

Comment:
Please strongly consider using `for (auto & fabric : chip::Server::GetInstance().GetFabricTable())` instead of manual iterator bits.